hack/ship-it-dev.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kind.sigs.k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  name: ship-it-dev
+nodes:
+  - role: control-plane
+    extraPortMappings:
+      - hostPort: 31901
```

Comment:
This'll break if the nodePort changes :( not sure if there's a good way to keep them in-sync. Not too big a deal though

Reply:
Yeah this is OK for now.